Automattic/vip-go-mu-plugins
# Set width and height for Block Editor images

## Description

Fixes #1391 by adding the correct `width` and `height` attributes to images added to post content using the block editor. Needs proper testing and consideration. cc @brandon-m-skinner: as mentioned in BB8-6850.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Outline the steps to test and verify the PR here.

1. Enable the block editor with `add_filter( 'use_block_editor_for_post', '__return_true'  );`
1. Create a new post
1. Add an image block, choosing an image from the media library that has dimensions wider than a `large` image
1. In the block inspector, [choose a display size](https://d.pr/i/T8VZix) other than `Full Size`
1. Publish the post and observe that the `width` and `height` attributes on the `<img>` element are those of the full size image.
1. Apply this PR
1. Reload the browser and observe that the dimensions reflect those of the size chosen in the block attributes.
